Sources/NIOExtras/LengthFieldPrepender.swift

```diff
@@ -13,6 +13,27 @@
 //===----------------------------------------------------------------------===//
 
 import NIO
+import Foundation
```

Comment:
NIO packages don't usually depend on `Foundation` so they can be used where that's not appropriate. Any particular reason to `import Foundation` here?